extensions/runtimes/src/index.ts

```diff
@@ -456,10 +500,15 @@ export default async (composer: any): Promise<void> => {
     name: 'C# - Functions',
     // startCommand: 'dotnet run',
     // path: dotnetTemplatePath,
-    build: async (runtimePath: string, _project: any) => {
-      composer.log(`BUILD THIS C# PROJECT! at ${runtimePath}...`);
+    build: async (runtimePath: string, _project: any, fullSettings?: any, port?: string) => {
```

Comment:
nit: If both of these new args always appear together, it's better to put them under one single optional arg.
It will have the added benefit of named args.

p.s: seems like project is prefixed with _ which perhaps means that it's not used.

Reply:
I updated the _projects to remove the underscore where the parameter is now in play